dbt/models/default/default.vw_pin_status.sql

```diff
@@ -0,0 +1,160 @@
+        consider EX or vacant land in oby.class to determine whether a parcel is
+        residential or commercial. */
+        MAX(
+            CASE
+                /* If a parcel exists in dweldat and comdat, or dweldat and oby
+                and class in oby is not vacant land, exempt, or class 2, then
+                it's mixed use. */
+                WHEN
+                    dwel.class IS NOT NULL
+                    AND (
+                        com.parid IS NOT NULL
+                        OR (
+                            oby.class IS NOT NULL
+                            AND (SUBSTR(oby.class, 1, 1) NOT IN ('1', '2'))
+                            AND oby.class NOT IN ('OA2', 'EX')
+                        )
+                    )
+                    THEN TRUE
+                    -- If a parcel is class 2 in oby and exists in comdat, then
+                    -- it's mixed use.
+                WHEN
+                    (SUBSTR(oby.class, 1, 1) = '2' OR oby.class = 'OA2')
+                    AND com.class IS NOT NULL
+                    THEN TRUE
+                ELSE FALSE
+            END
+        ) AS is_mixed_use
+    FROM {{ source('iasworld', 'pardat') }} AS par
+    LEFT JOIN {{ source('iasworld', 'comdat') }} AS com
+        ON par.parid = com.parid
+        AND par.taxyr = com.taxyr
+        AND com.cur = 'Y'
+        AND com.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'dweldat') }} AS dwel
+        ON par.parid = dwel.parid
+        AND par.taxyr = dwel.taxyr
+        AND dwel.cur = 'Y'
+        AND dwel.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'oby') }} AS oby
+        ON par.parid = oby.parid
+        AND par.taxyr = oby.taxyr
+        AND oby.cur = 'Y'
+        AND oby.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'land') }} AS land
+        ON par.parid = land.parid
+        AND par.taxyr = land.taxyr
+        AND land.cur = 'Y'
+        AND land.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'aprval') }} AS aprval
+        ON par.parid = aprval.parid
+        AND par.taxyr = aprval.taxyr
+        AND aprval.cur = 'Y'
+        AND aprval.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'exadmn') }} AS admn
+        ON par.parid = admn.parid
+        AND par.taxyr = admn.taxyr
+        AND admn.cur = 'Y'
+        AND admn.deactivat IS NULL
+    WHERE par.cur = 'Y'
+        AND par.deactivat IS NULL
+    GROUP BY par.parid, par.taxyr
+),
+
+-- These CTEs make it easier to work with CDUs from oby, comdat, dweldat since
+-- those tables aren't unique by parid and taxyr.
+dwel_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'dweldat'),
+    cdu_column = 'cdu'
+    ) }}),
+
+com_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'comdat'),
+    cdu_column = 'user16'
+    ) }}),
+
+oby_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'oby'),
+    cdu_column = 'user16'
+    ) }})
```

Comment:
**nitpick:** Do these still need to be CTEs? Maybe just use them inline and use positional arguments to shorten things a bit.

Reply:
**issue (blocking):** Also, it looks like these columns are null... At least I don't see any values for the `dwel_cdu` in 2023 or 2024.

Reply:
lowercase `cur = 'y'` in the macro...